src/main/java/com/cleanroommc/groovyscript/compat/mods/actuallyadditions/Empowerer.java

```diff
@@ -184,6 +184,12 @@ public RecipeBuilder blue(float blue) {
             return this;
         }
 
+        @Override
+        protected int getMaxItemInput() {
+            // More than 1 item cannot be placed on the Empowerer or Display Stands
+            return 1;
```

Comment:
the code here is a bit odd because it works with `mainInput` + `input.size() == 4` OR `input.size() == 5`. however, that is applied in the validate step prior to `validateItems` so you need to add something like
`msg.add(IngredientHelper.overMaxSize(mainInput, 1), "The mainInput must have a stack size of 1");`

Reply:
oops. I did find most of the "multiple input data lists" but not this apparently.